code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -50,6 +50,10 @@
 		to_chat(user, "<span class='notice'>You place [I] into [src] to start the fermentation process.</span>")
 		addtimer(CALLBACK(src, .proc/makeWine, fruit), rand(80, 120) * speed_multiplier)
 		return TRUE
+	var/obj/item/W = I
+	if(W)
+		if(W.is_refillable())
```

Comment:
I is already typed to obj/item

Reply:
😭😭 i forgot ab this maybe do it tomorrow